ModelicaTest/Electrical/QuasiStatic/SinglePhase.mo

```diff
@@ -265,5 +265,6 @@ package SinglePhase "Single-phase quasi-static package"
     connect(currentCurrent.pin_p, impedanceC.pin_n) annotation (Line(points={{20,-50},{20,-60},{50,-60},{50,-50}}, color={85,170,255}));
     connect(groundI1.pin, currentCurrent.pin_p) annotation (Line(points={{0,-60},{20,-60},{20,-50}}, color={85,170,255}));
     connect(complexRampC.y, impedanceC.Z_ref) annotation (Line(points={{69,-40},{62,-40}}, color={85,170,255}));
+  annotation(experiment(StopTime = 1.0, Interval = 0.0002));
```

Comment:
```suggestion
  annotation(experiment(StopTime = 1.0));
```
I don't see a need for Interval, as the other examples in the same package doesn't have it and still has the same number of outputs, and there is nothing interesting in these signals at that time-scale. I guess the added points are added to make the references "better" in some way.

Reply:
Ditto

Reply:
In general you're right, for QuasiStatic models we don't need an interval as long as no ramps are included.
I don't see which of the Test Examples is meant. 
If we use a ramp, we should consider specifying the Interval to see a meaningfull count of result points during the ramp (maybe 100 or 1000, 5000 is a little bit too much).

Reply:
I meant ModelicaTest.Electrical.QuasiStatic.SinglePhase.SerialConnection and ModelicaTest.Electrical.QuasiStatic.SinglePhase.Ideal - and the outputs is just smooth.